### PR TITLE
Add web service

### PR DESCRIPTION
**Pull request by kawai924**

### CODE REVIEW

File: entrypoint.py [hunk 1]
```diff
@@ -63,6 +67,19 @@ def execute_manual_query():
 # Press the green button in the gutter to run the script.
 if __name__ == '__main__':
     # comment out specific line if only project 2 or 3 should run
-    # create_rdf()  # Project 2
+    # # create_rdf()  # Project 2
```
Comment (AndiSap) [inline entrypoint.py:70]: Can you change back to just one #

File: entrypoint.py [hunk 2]
```diff
@@ -63,6 +67,19 @@ def execute_manual_query():
...
     execute_queries()  # Project 3 for all automated queries
-    # execute_manual_query()  # Project 3 for executing manual queries aka search engine
+    # # execute_manual_query()  # Project 3 for executing manual queries aka search engine
```
Comment (AndiSap) [inline entrypoint.py:72]: Same here

File: query/borough_accident.py [hunk 3]
```diff
@@ -69,20 +66,18 @@ def start():
     # displaying the DataFrame
     # print(tabulate(df, headers='keys', tablefmt='psql', showindex=False))
 
-    with open(output_location + 'greatest-accident-borough.txt', 'w') as f:
-        f.write(
-            ' Which borough in NY had the greatest number of accidents due to view obstruction in heavy fog? ---- Answer: ' +
-            str(df['Borough'].value_counts().idxmax()) + '\n\n')
-        f.write(tabulate(df, headers='keys', tablefmt='psql', showindex=False))
+    # with open(output_location + 'greatest-accident-borough.txt', 'w') as f:
```
Comment (AndiSap) [inline query/borough_accident.py:69]: Think we can remove this

**Review by AndiSap — COMMENTED** [review]
Some minor stuff, otherwise looks good

File: query/borough_accident.py (hunk 3 above)
Comment (kawai924) [reply query/borough_accident.py]: done

**Review by AndiSap — APPROVED** [review]

**Pull request merged**